apps/client/src/components/Question.js

```diff
@@ -146,6 +149,7 @@ Question.propTypes = {
   className: PropTypes.string,
   headingAs: PropTypes.string,
   hideNavigation: PropTypes.bool,
+  questionNeedsContactExit: PropTypes.bool,
```

Comment:
Would be nice if we could convert `Question.js` to `.ts` instead of using propTypes. Not sure how much work this is.

Reply:
Good point, I will add this to the refactor list.